extensions/wikia/UserRenameTool/RenameUserHelper.class.php

```diff
@@ -33,8 +33,8 @@ static public function lookupRegisteredUserActivity( $userID ) {
 		$result = [];
 		if ( empty( $wgDevelEnvironment ) ) { // on production
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)